### DISCUSSION TITLE
What's the reason behind making h-tags the same size? Potential bug

### DISCUSSION THREAD

**Discussion by DavidIsrawi** [tailwindlabs/tailwindcss#6627] [General]
**Potential Issue**

Is there a reason we are overriding the sizes of h1, h2, ... to be the same?

```
/* tailwindcss/src/css/preflight.css */

/*
Remove the default font size and weight for headings.
*/

h1,
h2,
h3,
h4,
h5,
h6 {
  font-size: inherit;
  font-weight: inherit;
}

```

Just curious as to why is this the case, I couldn't find an explanation on the sources I found.

**Potential Solution**

- My one thought is that we might want to keep away from this in order to use the Tailwind font-size property, but we can always assign a pre-determined em scale to the h-tags so the user doesn't have to manually do it. 

**Notes**

- Ultimately, the proper use of h-tags leads to better accessibility, SEO, and a bunch of other benefits. So I think promoting the use of them by providing less friction to leveraging the h-tags can be a good design call

Would love to hear other people's thoughts

**Comment by adamwathan** [comment]
Hey! This is explained in the documentation: https://tailwindcss.com/docs/preflight#headings-are-unstyled

**Reply by DavidIsrawi** [reply]
Woop - I missed that article. Thank you for the info!

**Comment by owen2345** [comment]
Should be configurable to decide whether to include or not.

**Reply by phenomen** [reply]
You can already disable resets by not importing `@tailwind base`

**Reply by wongjn** [reply]
You shouldn't omit `@tailwind base`. Rather, consider [disabling preflight](https://tailwindcss.com/docs/preflight#disabling-preflight) instead.
